### DISCUSSION TITLE
1st instances, illinois

### DISCUSSION THREAD

**Discussion by legaltextai** [freelawproject/courtlistener#3166] [Q&A]
I am trying to set up a research about the impact of covid in 2020-2022 in illinois. i take it there is no 1st instances  / state  in the database, only appeal, correct?

**Comment by mlissner** [comment]
Well, we have a lot of federal district court content, some of it from IL. Can you say what jurisdiction/content you're looking for?

**Reply by legaltextai** [reply]
my intuition is that those covid related cases , employment termination, doctor malpractice, benefits, etc, would fall under state jurisdiction, so it would be interesting to research these courts https://www.illinoiscourts.gov/courts-directory/interactive-map/by-court-type

**Reply by mlissner** [reply]
Yeah, we scrape the decisions from the Illinois Supreme Court and the Appellate Court, but we don't get filings from those cases.
